tomoni/theme.json

```diff
@@ -303,8 +249,22 @@
 							]
 						}
 					],
-					"fontFamily": "Noto Sans, Noto Sans JP",
```

Comment:
I think we need to keep this reference to Noto Sans JP. Otherwise, Noto Sans JP will never be used as `var(--wp--preset--font-family--noto-sans)` is just `Noto Sans`.

Reply:
You're right! I restored it: [25abb58](https://github.com/Automattic/themes/pull/7168/commits/25abb58b6686023682c35771bae4b211faf3327e)

For context, I removed it because I thought it was strange to have the fontFamily listed like this: 

<img width="1507" alt="Screenshot 2023-06-29 at 12 51 12 PM" src="https://github.com/Automattic/themes/assets/5375500/e4e35fcd-1cfb-4fb8-9449-1cd1fe9ba49e">

But I agree it's better to have the fallback available, so you don't have set a different fontFamily when using English and Japanese together.

Reply:
Oh, I see. It's a little weird, but it's also not too bad either.

Thanks for the performance improvement!